app/src/main/kotlin/com/wire/android/ui/home/settings/account/MyAccountScreen.kt

```diff
@@ -57,24 +75,40 @@ private fun mapToUISections(state: MyAccountState): List<AccountDetailsItem> {
 fun MyAccountContent(
     accountDetailItems: List<AccountDetailsItem> = emptyList(),
     forgotPasswordUrl: String?,
-    onNavigateBack: () -> Unit = {},
+    checkPendingSnackBarMessages: () -> SettingsOperationResult = { SettingsOperationResult.None },
+    onNavigateBack: () -> Unit = {}
 ) {
     val context = LocalContext.current
+    val snackbarHostState = remember { SnackbarHostState() }
+    LaunchedEffect(Unit) {
+        val result = checkPendingSnackBarMessages()
```

Comment:
Are we sure that once this message is shown we won't be showing it again if this screen recomposed, i.e. sending the app to background and resuming it? 

I know we can't use the approach of collecting and displaying the snackbar messages on a shared flow because the `MyAccountScreen` might not be visible at that point, but I am just worried that we end up seeing that message coming up multiple times.

Reply:
Yup, I was wondering the same while developing, but at least I verified manually multiple scenarios, and behaves as it should. Also, the approach was taken from other part of the app, where we are doing a similar flow :D

Reply:
Yes, I saw that on the `GroupConversationDetailsScreen` but I think the main consense that we agreed on previous collectives was to slowly migrate to that treating the snackbar messages as one time consumed type of events. 

But since this case implies back navigation to previous screen, I assume it will not be that easy to implement. Right @saleniuk ?